src/components/HomepageCards/index.tsx

```diff
@@ -36,8 +36,8 @@ const CardList: CardItem[] = [
     link: "/category/concepts",
     description: (
       <>
-        Check out some general concepts that will help you understand how
-        EthSigner works under the hood and architected.
+        Check out some general concepts that will help you understand how gnark
+        works under the hood and architected.
```

Comment:
missing "is" architected I guess. Will update the text of the 3 cards on the landing page in a separate PR 👍

Reply:
I'll leave the 3 cards up to the next PR since you guys will know what's best to fill there, for now have fixed the grammar issue